PROJECT

```diff
@@ -20,4 +20,12 @@ resources:
   kind: SecurityPolicy
   path: github.com/vmware-tanzu/nsx-operator/pkg/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
```

Comment:
If you don't add the controller in this patch, `controller: true` shouldn't exist.

Reply:
ok